src/app/pages/addons/MarkerWithLabelExample.js

```diff
@@ -0,0 +1,49 @@
+/* global google */
+
+import {
+  default as React,
+  Component,
+} from "react";
+
+import {
+  withGoogleMap,
+  GoogleMap,
+  Marker,
+} from "../../../lib";
+
+import styles from "./markerWithLabel.css"
```

Comment:
These styles aren't directly referenced in the code, you can just use:
`import "./markerWithLabel.css";`